src/main/java/auth/controller/HomeController.java

```diff
@@ -12,4 +12,9 @@
 	public String homePage(Model model) {
 		return "home";
 	}
+	
+	@RequestMapping(value = { "/welcome" })
```

Comment:
This should be moved to WelcomeController. In general is not a best practice to mix secured resources with unsecured resources under same controller.